### ISSUE TITLE
Filters to be applied in order to specify how a game is related to another one must be an implementation detail

### ISSUE THREAD

**Issue by juanmbellini** [juanmbellini/PowerUp#35]
https://github.com/juanmbellini/PowerUp/blob/80f700b355a0d580b4b37d523d9d81ca00766ec0/paw-webapp/interfaces/src/main/java/ar/edu/itba/paw/webapp/interfaces/GameService.java#L38

The ```Set<FilterCategory> filters``` param should not be included in the method signature, as the service implementation should define how a game is related to another one (instead of the upper level, which is calling the method setting this param).

**Comment by lipusal** [comment]
No me parece que esté mal esto, sino tendrías que tener 3 implementaciones distintas del service para que cada uno devuelva juegos relacionados de una manera distinta. Es algo así como pasarle un `Comparator` a una colección ordenada, vos definís el orden. Anyway, esto es low priority así que no le demos bola por ahora

**Comment by juanmbellini** [comment]
Claro, pero lo que digo es que no es tarea del controller definir cómo se filtra, sino que debería ser tarea del service.
Yo lo que cambiaría es que la firma del método sea ```Collection<Game> findRelatedGames(long gameId)```, pero sólo en ```GameService```. En ```GameDao``` lo dejaría como está.
